charts/kube-otel-stack/values.yaml

```diff
@@ -244,9 +251,190 @@ metricsCollector:
       pipelines:
         metrics:
           receivers: [prometheus, otlp]
-          processors: [memory_limiter, resource, resourcedetection/env, k8sattributes, metricstransform/k8sservicename, batch]
+          processors:
+            [
+              memory_limiter,
+              resource,
+              resourcedetection/env,
+              k8sattributes,
+              metricstransform/k8sservicename,
+              batch,
+            ]
           exporters: [otlp]
 
+logsCollector:
+  name: logs
+  clusterName: ""
+  image: otel/opentelemetry-collector-contrib:0.76.3
+  enabled: false
+  mode: daemonset
+  resources:
+    limits:
+      cpu: 250m
+      memory: 250Mi
+    requests:
+      cpu: 250m
+      memory: 250Mi
+  env:
+    - name: LS_TOKEN
+      valueFrom:
+        secretKeyRef:
+          key: LS_TOKEN
+          name: otel-collector-secret
+  volumeMounts:
+    - mountPath: /var/log
+      name: varlog
+      readOnly: true
+    - mountPath: /var/lib/docker/containers
+      name: varlibdockercontainers
+      readOnly: true
```

Comment:
do we need any special RBAC to make this work?

Reply:
None as far as I can tell. This works as-is in GKE and AKS. I was actually surprised that there wasn't a special permission needed to read from the node disk.